### PR TITLE
Local network feature

### PR DESCRIPTION
**Pull request by pushpak1300**
**Added  Local network feature from** 
https://github.com/laravel/valet/pull/439
and
Resolves https://github.com/cpriego/valet-linux/issues/257

### CODE REVIEW

File: cli/valet.php [hunk 1]
```diff
@@ -307,17 +307,7 @@
             warning('Updating now...');
             passthru($script . ' update');
         }
-    })->descriptions('Update Valet Linux and clean up cruft');
-
-    /**
-     * Change the PHP version to the desired one.
-     */
-    $app->command('use [preferedversion]', function ($preferedversion = null) {
```
Comment (jmarcher) [inline cli/valet.php]: We need to keep this command

File: cli/valet.php [hunk 2]
```diff
@@ -307,45 +307,7 @@
...
-
-    /**
-     * Removes the fallback site path used by Valet.
-     */
-    $app->command('unfallback', function () {
-        Configuration::updateKey('fallback', null);
-        info('Your Valet fallback path has been removed.');
-    })->descriptions('Remove the fallback site path for uncaught urls');
-
+    })->descriptions('Determine if this is the latest version of Valet');
```
Comment (jmarcher) [inline cli/valet.php]: This is wrong here

Comment (pushpak1300) [reply cli/valet.php]: Fixed!

File: cli/valet.php (hunk 1 above)
Comment (jmarcher) [reply cli/valet.php]: @pushpak1300 Leave `cli/valet.php` as in `origin` no changes needed there.

**Pull request closed without merging**